### PR TITLE
Mark custom asserts as helpers for nicer line numbers in tests

### PR DESCRIPTION
**Pull request by ashb**
> Helper marks the calling function as a test helper function. When
> printing file and line information, that function will be skipped.

Go 1.9 added this, so it's safe to use

### CODE REVIEW

**Review by VinGarcia — APPROVED** [review]

**Comment by VinGarcia** [comment]
Approved, as soon as the tests pass feel free to merge it =]

**Comment by ashb** [comment]
@VinGarcia Appreciate it -- I don't have permissions to hit the merge button though 😁

**Comment by VinGarcia** [comment]
Oh weird, I thought you would after I had approved it xP

**Pull request merged**
